### PR TITLE
[FLINK-34741][docs] Translate "get-started" Page for Flink CDC Chinese Documentation.

### PR DESCRIPTION
**Pull request by loserwang1024**
Translate https://github.com/apache/flink-cdc/tree/master/docs/content/docs/get-started pages into Chinese.

### CODE REVIEW

**Comment by loserwang1024** [comment]
@lvyanquan , @PatrickRen , @leonardBang , CC

File: docs/content.zh/docs/get-started/introduction.md [hunk 1]
```diff
@@ -71,27 +66,25 @@ pipeline:
   parallelism: 2
 ```
 
-By submitting the YAML file with `flink-cdc.sh`, a Flink job will be compiled
-and deployed to a designated Flink cluster. Please refer to [Core Concept]({{<
-ref "docs/core-concept/data-pipeline" >}}) to get full documentation of all
-supported functionalities of a pipeline.
+通过使用 `flink-cdc.sh` 提交 YAML 文件，一个 Flink 作业将会被编译并部署到指定的 Flink 集群。
+By submitting the YAML file with `flink-cdc.sh`, 请参考 [核心概念]({{<
+ref "docs/core-concept/data-pipeline" >}}) 以获取 Pipeline 支持的所有功能的完整文档说明。
```
Comment (laglangyue) [inline docs/content.zh/docs/get-started/introduction.md:71]: Here are some errors with some not translated words.

Comment (loserwang1024) [reply docs/content.zh/docs/get-started/introduction.md]: Thanks, I forget to delete this. Done it now.

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md [hunk 2]
```diff
@@ -104,95 +101,96 @@ Then `exit` exits and creates the Doris Docker cluster.
...
 
    {{< img src="/fig/mysql-doris-tutorial/doris-create-table.png" alt="Doris create table" >}}
 
-## Submit job using FlinkCDC cli
-1. Download the binary compressed packages listed below and extract them to the directory ` flink cdc-3.0.0 '`：    
-   [flink-cdc-3.0.0-bin.tar.gz](https://github.org/apache/flink/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz)
-   flink-cdc-3.0.0 directory will contain four directory `bin`,`lib`,`log`,`conf`.
+## 通过 FlinkCDC cli 提交任务
+1. 下载下面列出的二进制压缩包，并解压得到目录  ` flink cdc-3.0.0 '`：    
+   [flink-cdc-3.0.0-bin.tar.gz](https://github.org/apache/flink/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz).
```
Comment (lvyanquan) [inline docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:184]: I found that this download link is invalid, can you update and fix it in English version?
The correct address is:
https://github.com/ververica/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md [hunk 3]
```diff
@@ -104,95 +101,96 @@ Then `exit` exits and creates the Doris Docker cluster.
...
+   flink-cdc-3.0.0 下会包含 `bin`、`lib`、`log`、`conf` 四个目录。
 
-2. Download the connector package listed below and move it to the `lib` directory  
-   **Download links are available only for stable releases, SNAPSHOT dependencies need to be built based on master or release branches by yourself.**
+2. 下载下面列出的 connector 包，并且移动到 `lib` 目录下
+   **下载链接只对已发布的版本有效, SNAPSHOT 版本需要本地基于 master 或 release- 分支编译.**
     - [MySQL pipeline connector 3.0.0](https://repo1.maven.org/maven2/org/apache/flink/flink-cdc-pipeline-connector-mysql/3.0.0/flink-cdc-pipeline-connector-mysql-3.0.0.jar)
     - [Apache Doris pipeline connector 3.0.0](https://repo1.maven.org/maven2/org/apache/flink/flink-cdc-pipeline-connector-doris/3.0.0/flink-cdc-pipeline-connector-doris-3.0.0.jar)
```
Comment (lvyanquan) [inline docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:190]: this download link is invalid too.

File: docs/content.zh/docs/get-started/quickstart/mysql-to-starrocks.md [hunk 4]
```diff
@@ -24,138 +24,135 @@ specific language governing permissions and limitations
...
-    - [StarRocks pipeline connector 3.0.0](https://repo1.maven.org/maven2/org/apache/flink/flink-cdc-pipeline-connector-starrocks/3.0.0/flink-cdc-pipeline-connector-starrocks-3.0.0.jar)
+## 通过 FlinkCDC cli 提交任务
+1. 下载下面列出的二进制压缩包，并解压得到目录 `flink-cdc-3.0.0`：    
+   [flink-cdc-3.0.0-bin.tar.gz](https://github.com/ververica/flink-cdc-connectors/releases/download/release-3.0.0/flink-cdc-3.0.0-bin.tar.gz)
+   flink-cdc-3.0.0 下会包含 bin、lib、log、conf 四个目录。
+
+2. 下载下面列出的 connector 包，并且移动到 lib 目录下  
+   **下载链接只对已发布的版本有效, SNAPSHOT 版本需要本地基于 master 或 release- 分支编译**
+   - [MySQL pipeline connector 3.0.0](https://repo1.maven.org/maven2/com/ververica/flink-cdc-pipeline-connector-mysql/3.0.0/flink-cdc-pipeline-connector-mysql-3.0.0.jar)
+   - [StarRocks pipeline connector 3.0.0](https://repo1.maven.org/maven2/com/ververica/flink-cdc-pipeline-connector-starrocks/3.0.0/flink-cdc-pipeline-connector-starrocks-3.0.0.jar)
```
Comment (lvyanquan) [inline docs/content.zh/docs/get-started/quickstart/mysql-to-starrocks.md:152]: Invalid link too.

File: docs/content.zh/docs/get-started/introduction.md [hunk 5]
```diff
@@ -24,28 +24,23 @@ specific language governing permissions and limitations
...
 
-Flink CDC is a streaming data integration tool that aims to provide users with
-a more robust API. It allows users to describe their ETL pipeline logic via YAML
-elegantly and help users automatically generating customized Flink operators and
-submitting job. Flink CDC prioritizes optimizing the task submission process and
-offers enhanced functionalities such as schema evolution, data transformation,
-full database synchronization and exactly-once semantic.
+Flink CDC 是一个基于流的数据集成工具，旨在为用户提供一套功能更加全面的编程接口（API）。
+该工具使得用户能够以 YAML 配置文件的形式，优雅地定义其 ETL（Extract, Transform, Load）流程，并协助用户自动化生成定制化的 Flink 算子并且提交 Flink 作业。
+Flink CDC 在任务提交过程中进行了优化，并且增加了一些高级特性，如表结构变更自动同步（Schema Evolution）、数据转换（Data Transformation）、全量数据库同步（Full Database Synchronization）以及 Exactly-once 语义。
```
Comment (laglangyue) [inline docs/content.zh/docs/get-started/introduction.md:31]: just some suggestions
全量数据库同步 -> 整库同步
Exactly-once -> 精确一次（Exactly-once）

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md (hunk 2 above)
Comment (loserwang1024) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md]: Thanks, fix it.

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md (hunk 3 above)
Comment (loserwang1024) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md]: fix it

File: docs/content.zh/docs/get-started/quickstart/mysql-to-starrocks.md (hunk 4 above)
Comment (loserwang1024) [reply docs/content.zh/docs/get-started/quickstart/mysql-to-starrocks.md:152]: fix it

File: docs/content.zh/docs/get-started/introduction.md (hunk 5 above)
Comment (loserwang1024) [reply docs/content.zh/docs/get-started/introduction.md:31]: It seems better, done it

**Review by leonardBang — APPROVED** [review]
Thanks @loserwang1024 for the contribution and @laglangyue for the review, LGTM

**Pull request merged**